nuts/base_tests/napalm_ping.py

```diff
@@ -106,8 +106,8 @@ def napalm_ping_multi_dests(self, task: Task, **kwargs: Any) -> Result:
             if entry["host"] == task.host.name
         ]
         for destination in destinations_per_hosts:
-            result = task.run(task=napalm_ping, dest=destination, **kwargs)
-            result[0].destination = destination  # type: ignore[attr-defined]
+            result = task.run(task=napalm_ping, dest=destination, **kwargs)  # type: Any
```

Comment:
why did you change this annotation? the original was because mypy is nagging about `destination` because we patch it as property onto the class, so the original command instructed mypy to ignore it.

Reply:
I first did not really understand what was going on there, knowing it now does not make me happier though. Anyway I switched back to the original type annotation now.